config_files/survey_year_2019_config.yml

```diff
@@ -8,13 +8,22 @@
 # Parameters
 
 survey_year: 2019            # survey year being considered
-species: pacific_hake        # target species for the survey year
+species:
+  text_code: pacific_hake    # target species for the survey year -- species name
+  number_code: 22500         # target species for the survey year -- numeric code
 CAN_haul_offset: 200         # The value to be added to the Canadian's haul number
+sex_code:
+  Male: 
+    text_code: M
+    number_code: 1
+  Female:
+    text_code: F
+    number_code: 2
 
 ##############################################################################
 # Directory path that contains all input data needed
 
-data_root_dir: ../../2019_consolidated_files/
+data_root_dir: C:/Users/Brandyn/Documents/GitHub/EchoPro_data/2019_consolidated_files/
```

Comment:
You could change this to symbolic link, so that it always lives in the same place in the repo but could be linked to other places. In my local repo I have a symlink `EchoPro/data` that goes to some place where I put the test data.